test/testSwEngine.cpp

```diff
@@ -38,7 +38,7 @@ TEST_CASE("Basic draw", "[tvgSwEngine]")
     REQUIRE(canvas);
 
     uint32_t buffer[100*100];
-    REQUIRE(canvas->target(buffer, 100, 100, 100, SwCanvas::Colorspace::ABGR8888) == Result::Success);
+    REQUIRE(canvas->target(buffer, 100, 100, 100, SwCanvas::Colorspace::ABGR8888_STRAIGHT) == Result::Success);
```

Comment:
If we need a test of Colorspace::ABGR8888_STRAIGHT I think you can add it to testSwCanvas.